src/ansible_navigator/__init__.py

```diff
@@ -1,2 +1,14 @@
-"""The ansible-navigator application."""
-from ._version import __version__  # noqa: F401
+"""The ansible-navigator application.
+
+This is a transitional configuration during the migration
+from the hardcoded __version__ in _version to
+version being populated during the build.
+"""
+
+try:
+    from ._version import __version__
+except ImportError:
```

Comment:
Could you move this complexity into `_version` per #553?

Reply:
But I don't think we need to add the complexity now only to remove it later.

Reply:
You are assuming there will be a scm_version file, which has not been added to any PR or discussed. 

Also- I find the justification pointing to an issue that is being updated in the background a little awkward.
![image](https://user-images.githubusercontent.com/18386516/154995924-1915550e-a4c0-4eec-b3c4-121437bd1afe.png)

Reply:
I updated the issue because I realized it didn't have enough details and I didn't want to link the PR discussions because they tend to become hidden and hard to find on updates :man_shrugging:

Reply:
FWIW I don't think this change is reasonable. And `version` var also doesn't exist so I don't see any point in using it as a fallback. The reason I wanted a separate module first is that this technique has been tried out in a few projects and corner cases have been addressed.

Reply:
Also, this whole fallback dance should be happening in a dedicated Python module, not in `__init__`.

Reply:
But version _will_ exist, this separates the changes to the build from code changes.  when this goes in, the next PR can entirely be a change to the pyproject.toml file (I think)

Reply:
> the next PR can entirely be a change to the pyproject.toml file (I think)

That change will have to be coupled with integrating the SCM-generated file into the project. Otherwise, it won't be really atomic.

Reply:
> But version _will_ exist, this separates the changes to the build from code changes.

This still does not address the need to keep the detection code out of `__init__.py`.